src/components/SettingsPane/SettingsPane.stories.tsx

```diff
@@ -0,0 +1,58 @@
+import { Button, Chip, Link, Stack, Switch, Typography } from '@mui/material';
+import type { Meta, StoryObj } from '@storybook/react';
+import { SettingsPane, SettingsPaneItem } from '.';
+import { Code } from '../Code';
+
+const meta = {
+	title: 'Layout/Settings Pane',
+	component: SettingsPane,
+	tags: ['autodocs'],
+} satisfies Meta<typeof SettingsPane>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+	args: {
+		title: 'Account Settings',
+		children: (
+			<>
+				<SettingsPaneItem
+					title="Username"
+					description={
+						<Stack gap={2}>
+							<Code sx={{ width: 'fit-content' }}>johndoe113</Code>
+							<Typography variant="body" component="p">
```

Comment:
That's the default variant, right? 
```suggestion
							<Typography component="p">
```